### ISSUE TITLE
changelog-header-message-msg has no effect in independent mode

### ISSUE THREAD

**Issue by StarpTech** [ghiscoding/lerna-lite#325]
### Describe the bug

Hi, according to https://github.com/ghiscoding/lerna-lite/tree/main/packages/version#--changelog-header-message-msg this setting will affect only the changelog file at the root of the repository, but in the independent mode, we don't have one.

### Expectation

Add the `--changelog-header-message-msg` to all changelog files.

### Reproduction

See above.

### Lerna config and logs

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
```
</p></details>

<details><summary>lerna-debug.log</summary><p>

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>

### Environment Info

```shell
| Executable        | Version |
| ----------------- | ------- |
| `lerna --version` | 1.11.1 |
| `npm --version`   | VERSION |
| `yarn --version`  | VERSION |
| `node --version`  | VERSION |

---
OR simply run `npx lerna info` command and copy+paste the result here
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/lerna-lite/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/ghiscoding/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
yeah I didn't want to add too much data to all other changelogs but I didn't know that independent mode wasn't creating a changelog in the root. I can add it even though this will mean modifying a lot more changelogs

**Comment by ghiscoding** [comment]
both issues should be fixed under new release [v1.11.2](https://github.com/ghiscoding/lerna-lite/releases/tag/v1.11.2)

**Comment by StarpTech** [comment]
Thank you for the quick fix.
